parse/index.js

```diff
@@ -359,12 +360,17 @@ export default ({
 
     let data = []
     let index = 0
+    let currentAggregateData = null
```

Comment:
Not sure how much this adds but what about naming it `currentDataGroup` to distinguish it from the `aggregate` keyword?